api/envoy/config/metrics/v3/stats.proto

```diff
@@ -265,18 +266,18 @@ message TagSpecifier {
     //     }
     //   ]
     //
-    // The two regexes of the specifiers will be processed in the definition order.
+    // The two regexes of the specifiers will be processed from the elaborated
+    // stat name.
     //
-    // The first regex will remove ``ios.``, leaving the tag extracted name
-    // ``http.connection_manager_1.user_agent.downstream_cx_total``. The tag
-    // ``envoy.http_user_agent`` will be added with tag value ``ios``.
+    // The first regex will save ``ios.`` as thae tag value for envoy.http_user_agent``. It will
+    // leave it in the name for potential matching with additional tag specifiers. After all tag
+    // specifiers are processed the tags will be removed from the name.
     //
-    // The second regex will remove ``connection_manager_1.`` from the tag
-    // extracted name produced by the first regex
-    // ``http.connection_manager_1.user_agent.downstream_cx_total``, leaving
-    // ``http.user_agent.downstream_cx_total`` as the tag extracted name. The tag
-    // ``envoy.http_conn_manager_prefix`` will be added with the tag value
-    // ``connection_manager_1``.
+    // The second regex will populate tag ``envoy.http_conn_manager_prefix`` with value
+    // ``connection_manager_1.``, based on the original stat name.
+    //
+    // As a final step, the matched tags are removed. leaving
```

Comment:
nit: `.` may not be needed after `removed`? Also typo `thae` in L272

Reply:
thanks for the catch on the typo; fixed. Changed the "." to a ",".